web-share/README.md

```diff
@@ -1,14 +1,19 @@
 # Web Share Demo
 
+**Try it out:** https://scrapbook-pwa.web.app/
+
 This is a small [progressive web app](https://developers.google.com/web/progressive-web-apps/)
-demo that shows off the use of two emerging web platform features:
+demo that shows off the use of three emerging web platform features:
 
 - [Web Share Target API Level 2](https://developers.google.com/web/updates/2018/12/web-share-target),
 for receiving incoming media files shared **by** other installed applications.
 
 - [Web Share Level 2](https://developers.google.com/web/updates/2019/05/web-share-files),
 for taking local media files and natively sharing them **to** other installed applications.
 
+- [Content Indexing API](https://chromestatus.com/feature/5658416729030656),
```

Comment:
A WIP screenshot of Chrome Downloads would really help to understand quickly what this is about.

Reply:
I'm going to switch this link out to the web.dev article (which will have screenshots) once it's live. That will be in a separate PR.